html.go

```diff
@@ -859,8 +859,8 @@ func doubleSpace(out *bytes.Buffer) {
 func isRelativeLink(link []byte) (yes bool) {
 	yes = false
 
-	// a tag begin with '#'
-	if link[0] == '#' {
+	// a tag begin with '#' or '.'
+	if link[0] == '#' || link[0] == '.' {
 		yes = true
 	}
```

Comment:
This func `isRelativeLink` has 3 separate cases that may result in setting `yes` to `true`. Why did you decide to group '.' together with the first test case that checks if tag begins with '#', instead of adding a separate test case? It doesn't seem they are related.